### ISSUE TITLE
MeshSwap imports an extra collection/group

### ISSUE THREAD

**Issue by ghost** [TheDuckCow/MCprep#72]
Hi, I have a problem.When I place the square, it places two.
![批注 2019-05-22 200113](https://user-images.githubusercontent.com/50913224/58173445-f90c8d00-7ccd-11e9-88fc-f7f12060fabe.png)

**Comment by TheDuckCow** [comment]
Hi there,

I should update the documentation as this is a known issue, and there is not much of a workaround. It's because 1) the addon imports the whole group/collection as is, so it appears in the scene and then 2) makes a copy or instance of this group at the location of the cursor (if meshswap spawning) or in each place in the world import (if using MeshSwap on a world object). It reuses the same group/collection for future placements of that object.

This is the same as how it works in Blender 2.7 actually, however there the "soruce" group (what you see as the collection named "Enchantment_Table" in your screenshot) is placed on layer 20 to put it out of the way.

Quick fix: click on the collection "Enchantment_Table" in the outliner so it is highlighted, and then press e (for "exclude") to effectively hide it without affected other instances of it. I actually tried implementing this or something similar in python code, but there weird artifacts that would make it confusing to the user so I opted to leave it just imported in the scene like this until I find a better fix.

Also worth calling out, this will be true for all group/collections from the MeshSwapper. Also note that the location of the source group will match the same location as in the source MeshSwap blend file.
